sources/HUBViewController.m

```diff
@@ -973,21 +973,26 @@ - (nullable HUBComponentWrapper *)componentWrapperFromCell:(HUBComponentCollecti
 
 - (CGFloat)calculateTopContentInset
 {
+    id<HUBViewControllerDelegate> delegate = self.delegate;
+
+    if (![delegate viewControllerShouldAutomaticallyManageTopContentInset:self]) {
```

Comment:
This means that if the delegate is `nil`, the view controller won't manage content inset, which is not the behavior we want (the default should be to automatically manage). So you'll need to `nil`-check the delegate first.